.gitignore

```diff
@@ -9,3 +9,4 @@
 /*.egg-info
 /htmlcov
 __pycache__
+.idea/
```

Comment:
You should add the `.idea` folder to your [local git excludes instead](https://help.github.com/en/articles/ignoring-files#explicit-repository-excludes).